interface/src/index.html

```diff
@@ -26,9 +26,10 @@
     <!-- d3.network library, written by Aaron Wang (aaronkw)
       FIXME: Ideally d3.network.js should be installed in vendor/ subdirectory,
       but since it is under active development, it is requested online for now.
-      -->
+
     <script src="https://rawgit.com/dongbohu/d3-network/master/d3.network.js"
             charset="utf-8"></script>
+    -->
```

Comment:
This specific `FIXME` is addressed by adding `d3.network.js` to our repository, so I'd recommend removing lines 26-32 entirely.

If you want to have the reminder to follow up on managing the workflow for incorporating our "fork" of Aaron's code as we develop, it should probably be tracked as an issue now. I have a few thoughts I could add as comments to the issue.

Reply:
I didn't remove it because I am not sure whether it is a good idea to include ``d3.network.js` in our repo. I will remove it if both of you agree.

Reply:
I agree, I made a similar comment about this.